static/css/budget.less

```diff
@@ -1,57 +1,78 @@
+/* ==========================================================================
+   cfgov-refresh
+   Budgeting pages at /budget/.
+   ========================================================================== */
+
 .budget_page-content {
-    
+
     h4 {
-        margin-top: @grid_gutter-width;
+        // Negative offset is to accommodate for the gap added by line-height.
+        margin-top: unit( ((@grid_gutter-width * 2) - 15) / @base-font-size-px, em);
     }
-    
-    ul {
-      .list__unstyled();
-      .list__links();
-      .list__icons();
-
-      li {
-          .list_item();
-          
-          a {
-              .list_link();
-              .jump-link();
-          }
-      }
-    }
-    
-    &.financial-report a, 
-    &.performance-plan a,
-    &.funding-request a span{
-        &:before {
-            .cf-icon();
-            content: "\e402";
-            margin-right: .5em;
-            color: @black;
+
+    &.financial-report,
+    &.performance-plan,
+    &.funding-request {
+        ul {
+            .list__links();
+            .list__icons();
+
+            li {
+                .list__links .list_item();
+                .list__icons .list_item();
+                margin-left: 0;
+
+                a {
```

Comment:
Does the `a` need to be nested in the `li`? It's implied by living under the `ul` and reduces specificity